todo_app/tests/test_e2e.py

```diff
@@ -33,41 +37,26 @@ def app_with_temp_board():
 
     # Tear down
     thread.join(1)
-    delete_trello_board(board_id)
-
-def create_trello_board():
-    # TODO Create a new board in Trello and return the id
-    url = "https://api.trello.com/1/boards"
+    # deepcode ignore WrongNumberOfArguments/test: <please specify a reason of ignoring this>
```

Comment:
I don't really know deepcode but Google tells me it's a clever linter. Any idea why it thinks this should receive arguments when the function definition below clearly doesn't take any?

Reply:
I think that was left over when I was trying to configure it, I've deleted this comment as I don't think its relevant.